js/templates/reviews/review.html

```diff
@@ -0,0 +1,73 @@
+<% if (ob.error) { %>
+  <p class="clrTErr txCtr"><i class="ion-alert-circled"></i> <%= ob.polyT('listingDetail.errors.loadReview', { id: ob.ratingId, error: ob.error }) %></p>
+<% } else { %>
+
+  <%
+    let title = ob.vendorSig.metadata.listingTitle ? ob.vendorSig.metadata.listingTitle : ob.vendorSig.metadata.listingSlug;
+    title = title ? title : ob.polyT('reputation.noTitle');
```

Comment:
If the title fails to load because it's an old review, the slug will be shown instead. That's more user friendly than showing "title not found" in my opinion.